src/lib/Containers/Conversation.tsx

```diff
@@ -87,12 +78,8 @@ export class Conversation extends React.Component<RelayProps, State> {
     // End of the list reached. Increase page size. Relay
     // will fetch only the required data to fill the new
     // page size.
-    if (item.key === this.state.messages.length - 1) {
-      this.props.relay.setVariables({ pageSize: this.state.messages.length + PAGE_SIZE }, readyState => {
-        if (readyState.done) {
-          debugger
-        }
-      })
+    if (item.key === this.props.me.conversation.messages.edges.length - 1) {
```

Comment:
I get that this was sufficient as a a temp workaround, but it should be replaced with a proper scroll handler. Or possibly by using a pre-built one such as https://github.com/expo/react-native-invertible-scroll-view.